docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -527,15 +481,133 @@ export default function AppLayoutDocsFooter(props) {
                 </Alert>
               )}
               <DialogActions>
-                <Button type="reset">{t('cancel')}</Button>
-                <Button type="submit" variant="contained">
+                <Button type="reset" size="small">
+                  {t('cancel')}
+                </Button>
+                <Button type="submit" variant="contained" size="small">
                   {t('submit')}
                 </Button>
               </DialogActions>
-            </Box>
+            </div>
           </form>
         </Collapse>
-      </Box>
+        <Divider />
+        {hidePagePagination ? null : (
+          <Stack direction="row" justifyContent="space-between" sx={{ my: 2 }}>
+            {prevPage !== null ? (
+              <Button
+                size="small"
+                component={Link}
+                noLinkStyle
+                prefetch={false}
+                href={prevPage.pathname}
+                {...prevPage.linkProps}
+                startIcon={<ChevronLeftIcon />}
+                sx={{ '& > span': { ml: -1, mr: 0.5 } }}
+              >
+                {pageToTitleI18n(prevPage, t)}
+              </Button>
+            ) : (
+              <div />
+            )}
+            {nextPage !== null ? (
+              <Button
+                size="small"
+                component={Link}
+                noLinkStyle
+                prefetch={false}
+                href={nextPage.pathname}
+                {...nextPage.linkProps}
+                endIcon={<ChevronRightIcon />}
+              >
+                {pageToTitleI18n(nextPage, t)}
+              </Button>
+            ) : null}
+          </Stack>
+        )}
+        <Divider />
+        <Stack
+          direction={{ xs: 'column', sm: 'row' }}
+          alignItems="center"
+          spacing={{ xs: 3, sm: 1 }}
+        >
+          <Stack direction="row" alignItems="center" spacing={1} sx={{ flexGrow: 1 }}>
+            <Link href={ROUTES.blog}>
+              <FooterLink>
+                Blog <ArrowOutwardRoundedIcon sx={{ fontSize: 14 }} />
+              </FooterLink>
+            </Link>
+            <Typography color="grey.500" fontSize={13}>
+              &bull;
+            </Typography>
+            <Link href={ROUTES.store}>
+              <FooterLink>
+                Store <ArrowOutwardRoundedIcon sx={{ fontSize: 14 }} />
+              </FooterLink>
+            </Link>
+            <Typography color="grey.500" fontSize={13}>
+              &bull;
+            </Typography>
+            <Link target="_blank" rel="noopener noreferrer" href="mailto:contact@mui.com">
```

Comment:
Good call!

Reply:
Decided to change it to "Careers" instead because the Support page living within the Material UI docs feels weird to me (if we're going to use it as a company-wide support policy). Simpler approach 👍